backend/apps/github/api/label.py

```diff
@@ -15,8 +15,8 @@ class Meta:
             "name",
             "description",
             "color",
-            "created_at",
-            "updated_at",
+            "nest_created_at",
+            "nest_updated_at",
```

Comment:
It seems we don't have that data. Just remove it, users don't need our internal timestamps.